logging/Cargo.toml

```diff
@@ -14,7 +14,21 @@ doctest = false
 testutils = ["soroban-sdk/testutils"]
 
 [dependencies]
-soroban-sdk = { workspace = true }
+soroban-sdk = "0.7.0"
 
 [dev_dependencies]
-soroban-sdk = { workspace = true, features = ["testutils"] }
+soroban-sdk = { version = "0.7.0", features = ["testutils"] }
+
+[profile.release]
```

Comment:
Can this be defined in the root Cargo.toml? Adding the profiles to every example seems too verbose.

Reply:
I just noticed that their is a root Cargo.toml and that it already has this profile.release. I will take this PR back to investigate why it was complaining.